drivers/adc/max9611pmb/max9611.h

```diff
@@ -0,0 +1,197 @@
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+#ifndef __MAX9611_H__
+#define __MAX9611_H__
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+#include <stdint.h>
+#include <stdlib.h>
+#include "no_os_i2c.h"
+#include "no_os_util.h"
+#include "no_os_error.h"
+#include "no_os_delay.h"
+
+/******************************************************************************/
+/************************** max538x Definitions *******************************/
+/******************************************************************************/
+#define MAX9611_MUX_MASK		NO_OS_GENMASK(2, 0)
+#define MAX9611_MODE_MASK		NO_OS_GENMASK(7, 5)
+
+#define MAX9611_SHDN_MASK		NO_OS_BIT(3)
+#define MAX9611_LR_MASK			NO_OS_BIT(4)
+#define MAX9611_DTIME_MASK		NO_OS_BIT(3)
+#define MAX9611_RTIME_MASK		NO_OS_BIT(2)
+
+#define MAX9611_OUT_OF_BOUNDS  	0x0C
+
+/******************************************************************************/
+/*************************** Types Declarations *******************************/
+/******************************************************************************/
+enum max9611_regs {
+	MAX9611_CSA_MSB,
+	MAX9611_CSA_LSB,
+	MAX9611_RSP_MSB,
+	MAX9611_RSP_LSB,
+	MAX9611_OUT_MSB,
+	MAX9611_OUT_LSB,
+	MAX9611_SET_MSB,
+	MAX9611_SET_LSB,
+	MAX9611_TMP_MSB,
+	MAX9611_TMP_LSB,
+	MAX9611_CTR1,
+	MAX9611_CTR2,
+};
+
+enum max9611_data {
+	MAX9611_DATA_CSA,
+	MAX9611_DATA_RSP,
+	MAX9611_DATA_OUT,
+	MAX9611_DATA_SET,
+	MAX9611_DATA_TMP,
+};
+
+enum max9611_addr_lvls {
+	MAX9611_ZERO_VCC,
+	MAX9611_33_VCC,
+	MAX9611_66_VCC,
+	MAX9611_100_VCC,
+};
+
+enum max9611_delay_time {
+	MAX9611_1MS,
+	MAX9611_100US,
+};
+
+enum max9611_retry_time {
+	MAX9611_50MS,
+	MAX9611_10MS,
+};
+
+enum max9611_mux_conf {
+	MAX9611_CONF_SENSE_1X,
+	MAX9611_CONF_SENSE_4X,
+	MAX9611_CONF_SENSE_8X,
+	MAX9611_CONF_IN_COM_MODE,
+	MAX9611_CONF_OUT_ADC,
+	MAX9611_CONF_SET_ADC,
+	MAX9611_CONF_TEMP,
+	MAX9611_FAST_MODE,
+};
+
+enum max9611_mode_conf {
+	MAX9611_NORMAL_MODE,
+	MAX9611_OPAMP_MODE = 0x3,
+	MAX9611_COMPARATOR_MODE = 0x7,
+};
+
+struct max9611_dev {
+	/* I2C */
```

Comment:
No need to specify this.

Reply:
Removed in latest commit for max9611_dev and max9611_init_param structs.